tests/unit/certGen.unitTest.ts

```diff
@@ -2093,6 +2094,462 @@ describe("cert-gen", () => {
         });
     });
 
+    context("CertGenService for ADR", () => {
+        context("when a passing test result for ADR is read from the queue", () => {
+            const event: any = lodash.cloneDeep(queueEventPass);
```

Comment:
`cloneDeep(queueEventPass);` will reduce the import size.

Reply:
don't quite understand what's the requirement here

Reply:
solved